src/reducers/cartReducer.js

```diff
@@ -0,0 +1,49 @@
+const cartReducer = (state, action) => {
+    switch (action.type) {
+		case "CART_REQUEST":
+			return (state = {
+                ...state,
+                loader: true
+            });
+		case "CART_SUCCESS":
+			return (state = {
+                ...state,
+                loader: false,
+                cartInfo: action.payload
+            });
+        case "CART_SUCCESS_MATCH":
+			return (state = {
+                ...state,
+                loader: false,
+                productAdded: true
+            });
+        case "CART_SUCCESS_MATCH_UNDO":
+            return (state = {
+                ...state,
+                loader: false,
+                productAdded: false
+            });
+        case "CART_SUCCESS_MATCH_UNDO":
```

Comment:
Repeated CART_SUCCESS_MATCH_UNDO case. Remove one.

Reply:
Thanks for the reminder.